### PR TITLE
Added failure message

### PR DESCRIPTION
**Pull request by Glimmereyes**
Added a failure message when pushing the boulder into Gate of Souls

### CODE REVIEW

File: bescort.lic [hunk 1]
```diff
@@ -831,18 +831,18 @@ class Bescort
...
     case waitfor(/^At the bottom of the hollow/, /^You stop pushing/, /blocking all access to a low tunnel\.$/)
-    when /You stop pushing/, /blocking all access to a low tunnel/
+    when /^You stop pushing/, /blocking all access to a low tunnel\.$/
       push_boulder
     end
   end
   
   def gos_tunnel
-    fput('kneel')
+    fput('kneel', 'You kneel down upon the ground')
```
Comment (rpherbig) [inline bescort.lic]: 💡 Can this use `bput` instead?

Comment (Sarvatt) [reply bescort.lic]: aka please consider switching it over in the future even though its merged.

Comment (Sarvatt) [reply bescort.lic]: you might have meant to switch it to bput there, fput doesnt take args like that

Comment (rpherbig) [reply bescort.lic]: `fput` does take args, similar to `bput`, it's just less reliable.

Comment (rpherbig) [reply bescort.lic]: https://github.com/rcuhljr/dr-lich/blob/master/lich.rbw#L5700

**Pull request merged**